diff --git a/index.js b/index.js
index bcb70193b4bf60c5a1d021a6756adf5f81d67ac6..aebe7d24cf2fb19f9aa483261126a3f3f46f51c6 100644
--- a/index.js
+++ b/index.js
@@ -5,7 +5,7 @@ const pify = require('pify')
 const stripBom = require('strip-bom')
 const yaml = require('js-yaml')
 
-const parse = data => yaml.safeLoad(stripBom(data))
+const parse = data => yaml.load(stripBom(data))
 
 const readYamlFile = fp => pify(fs.readFile)(fp, 'utf8').then(data => parse(data))
 
